error[E0433]: failed to resolve: could not find `ErrorKind` in `fs`
 --> /tmp/codegemma7b-file_size-go-rs.rs:8:44
  |
8 |         println!("{}", fmt::Error::new(fs::ErrorKind::NotFound));
  |                                            ^^^^^^^^^ could not find `ErrorKind` in `fs`
  |
help: consider importing this enum
  |
1 + use std::io::ErrorKind;
  |
help: if you import `ErrorKind`, refer to it directly
  |
8 -         println!("{}", fmt::Error::new(fs::ErrorKind::NotFound));
8 +         println!("{}", fmt::Error::new(ErrorKind::NotFound));
  |

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0433`.
